tests/basics/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that basic interactions work
+execute:
```

Comment:
Aha, thank you! I was totally confused by the unreasonable output this task was generating, and did not dig down as I was distracted making native kernels work. I wish this was invalid yaml, instead of being a curious form of strings. :(
```
>>> yaml.load("foo:\n  bar\n  baz")
{'foo': 'bar baz'}

>>> yaml.load("foo: |\n  bar\n  baz")
{'foo': 'bar\nbaz'}
>>> 
```